README.md

```diff
@@ -615,6 +615,12 @@ See [README.i18n.md](README.i18n.md).
 
 ## Using Tor
 
+> There is a new, still experimental, [integration with Tor](ANONYMITY_NETWORKS.md). The
+> feature allows connecting over IPv4 and Tor simulatenously - IPv4 is used for
```

Comment:
simultaneously

Reply:
Fixed.

Reply:
Yikes, just got merged. Will have to update this separately.